src/applications/mhv/secure-messaging/components/MessageActionButtons.jsx

```diff
@@ -6,7 +6,7 @@ import MoveMessageToFolderBtn from './MessageActionButtons/MoveMessageToFolderBt
 import PrintBtn from './MessageActionButtons/PrintBtn';
 import { DefaultFolders } from '../util/constants';
 import ActionButtons from './shared/ActionButtons';
-import ReplyButton from './MessageActionButtons/ReplyButton';
+import ReplyBtn from './MessageActionButtons/ReplyBtn';

```

Comment:
NITPICK: Unless I missed something, We should stick with the convention of using `Button` instead of `Btn`

Reply:
There was an existing "ReplyButton" that was used in the top row with "PrintBtn", etc.  So when I needed to put a reply button in the body and just show the top one in the sent folder, I renamed "ReplyButton" to "ReplyBtn" like the PrintBtn and then created a new "ReplyButton" for the body.   Wasn't sure about the conventions.